.test-infra/jenkins/CommonJobProperties.groovy

```diff
@@ -193,7 +193,7 @@ class CommonJobProperties {
     context.switches("-Dorg.gradle.vfs.watch=false")
 
     // Include dependency licenses when build docker images on Jenkins, see https://s.apache.org/zt68q
-    context.switches("-Pdocker-pull-licenses")
```

Comment:
This has no effect. Probably Jenkins uses the file in the master branch (or somewhere not specific to branches)